src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -165,5 +150,5 @@ find_in_ancestors() {
 
     # gnerate and build
```

Comment:
```suggestion
    # generate and build
```
Yes, I know, pre-existing.